contracts/round/RoundFactory.sol

```diff
@@ -27,24 +27,18 @@ contract RoundFactory is OwnableUpgradeable {
   // --- Data ---
 
   /// @notice Address of the RoundImplementation contract
-  address public roundContract;
+  address public roundImplementation;
 
-  /// @notice Address of the protocol treasury
-  address payable public protocolTreasury;
-
-  /// @notice Protocol fee percentage
-  uint8 public protocolFeePercentage;
+  /// @notice Address of the Allo settings contract
+  address public alloSettings;
```

Comment:
doesn't this change create problems with the fact that the factory is deployed as an upgradeable proxy?

Reply:
Only on goerli !
The other chains don't have these things deployed, so 
- on goerli we would deploy a fresh factory ( a clean slate)
- on mainnet/ fantom-testnet/fantom / optimism -> it would be a simple upgrade

Reply:
that's awesome thank you @thelostone-mc . We already talked about it but I confused it during the review, thank you! <3